### PR TITLE
Add support for WALA SDG and build a CLI tool for standalone usage

### PR DESCRIPTION
**Pull request by rahlk**

### CODE REVIEW

File: cargo/standalone.py
Comment (rofrano) [inline cargo/standalone.py:1]: Please add IBM Copyright to the top of this file.

File: cargo/utils/summarizer.py
Comment (rofrano) [inline cargo/utils/summarizer.py:1]: This file is empty. Is it needed?

File: cargo/utils/transform_graph.py
Comment (rofrano) [inline cargo/utils/transform_graph.py:1]: Please add IBM Copyright to this file. Also the first methods have empty docstrings. Just describe what the method does (even though the name is descriptive).

**Review by rofrano — CHANGES_REQUESTED** [review]
Looks great. Just add the missing copyrights and docstrings and you're good to go.

**Comment by rahlk** [comment]
Thanks JR, I ought to have marked this as a draft. This is still work in progress.

--
Rahul Krishna, Ph. D.,
Research Scientist, Hybrid Cloud,
IBM T. J. Watson Research Center.
W// rkrsn.us
E// ***@***.***

**Review by rahlk — COMMENTED** [review]
Done

**Pull request merged**